examples/PicardIterations_SAM-OpenMC_VHTR_Workbench/watts_comprehensive.son

```diff
@@ -0,0 +1,74 @@
+watts{
+    % Change the working directory to the directory where the input files are stored. This is necessary
+    % due to how WATTS copies input files to the temporary working directory.
+    workflow_dir = "/Users/zhieejhiaooi/Documents/ANL/watts-devel/watts/examples/PicardIterations_SAM-OpenMC_VHTR_Workbench"
+
+    plugins{
+        plugin(ID1){
+            code = moose
+            template = "sam_template"
+            exec_dir = "SAM_DIR"
+            exec_name = "sam-opt"
+            show_stderr = False
+            show_stdout = False
+        }
+        plugin(ID2){
+            code = openmc
+            template = "openmc_template.py"
+            scores = ["nu-fission"] % Name of scores for OpenMC tallies. This will be the base name used for saving the tallies.
+            score_names = ["Init_P"] % A more representative name for the scores
```

Comment:
it is not clear how `Init_P` will translate into ["Init_P_1" "Init_P_2" "Init_P_3" "Init_P_4" "Init_P_5"] - this is coming directly from OpenMC results I believe, so difficult to explain much better...

Reply:
Added a few lines to explain how the code processes `scores` and use `score_names`.